Connector CLI operations (#350)
nmshd/documentation: _docs_integrate/delete-identities.md

# End automatic generation
---

> It is not possible to utilize the use cases regarding the [IdentityDeletionProcesses]({% link _docs_integrate/data-model-overview.md %}#identitydeletionprocess) mentioned below as an Integrator of a Connector.

Reviewer: ```suggestion
It is not possible to utilize the use cases regarding the [IdentityDeletionProcesses]({% link _docs_integrate/data-model-overview.md %}#identitydeletionprocess) mentioned below using the REST API of the Connector.
```

below you write that it's possible as an integrator. 

Additionally I'd move the part HOW it's possible in the connector up here.
